src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/CodeActions/AddUsingsCodeActionProviderFactoryTest.cs

```diff
@@ -78,24 +78,39 @@ public void TryExtractNamespace_ReturnsTrue()
         var csharpAddUsing = "using Abc.Xyz;";
 
         // Act
-        var res = AddUsingsCodeActionProviderHelper.TryExtractNamespace(csharpAddUsing, out var @namespace);
+        var res = AddUsingsCodeActionProviderHelper.TryExtractNamespace(csharpAddUsing, out var @namespace, out _);
```

Comment:
Can we validate the out variable on these tests? (I think it will be `string.Empty`)